srcpkgs/rofi-calc/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'rofi-calc'
+pkgname=rofi-calc
+version=0.1
+revision=1
+build_style=gnu-configure
+hostmakedepends="automake libtool pkg-config"
+makedepends="rofi-devel libqalculate-devel"
```

Comment:
The package builds fine without `libqalculate-devel` but won't run without being able to execute the `qcalc` command. So I think you should drop `libqalculate-devel` from makedepends and instead put `qalculate` into depends. Wouldn't it also make sense to depend on rofi since the package is completely useless without it?

Reply:
Thanks, the readme stated it needs the libqalculate headers, so I did not check if it actually uses them. I changed the qalculate dependency and added rofi.